### PR TITLE
WIP: Windows support

### PR DESCRIPTION
**Pull request by richardotis**
Based on PR #8 started by @basnijholt

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,16 @@
+CD src
+
+copy Makefile.inc.i686_pc_mingw32 Makefile.inc
```
Comment (basnijholt) [inline recipe/bld.bat]: I think this is failing in the [CI](https://ci.appveyor.com/project/conda-forge/scotch-feedstock/build/1.0.11/job/8jlwu39uu0mjm9j6#L625).

Comment (richardotis) [inline recipe/bld.bat]: Yes, I'm having trouble figuring out how to get the patched Makefile in the right place.

**Comment by richardotis** [comment]
`nmake` doesn't support the `+=` syntax so this will probably require a custom Makefile

**Comment by richardotis** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Let’s close here - follow up cmake version is in #73

**Pull request closed without merging**